static/js/tippyjs.js

```diff
@@ -174,6 +175,22 @@ export function initialize() {
         e.currentTarget._tippy.destroy();
     });
 
+    delegate("body", {
+        target: "#message_type_icon",
+        appendTo: () => document.body,
+        onShow(instance) {
+            const $elem = $(instance.reference)[0];
+            instance.setContent(
+                $elem.classList.contains("is_pm")
```

Comment:
I'd call this class `is_private_message`; we avoid abbreviations where possible in the codebase.

Reply:
Nope, that was me :) There are other uses of `is_pm` in the codebase, but they're not related to this functionality.